core/network/src/protocol/sync.rs

```diff
@@ -28,11 +28,24 @@
 //!
 
 use blocks::BlockCollection;
-use client::{BlockStatus, ClientInfo, error::Error as ClientError};
-use consensus::{BlockOrigin, import_queue::{IncomingBlock, BlockImportResult, BlockImportError}};
+use client::{ClientInfo, error::Error as ClientError};
+use consensus::{
+	BlockOrigin,
+	BlockStatus,
+	block_validation::{BlockAnnounceValidator, Validation},
+	import_queue::{IncomingBlock, BlockImportResult, BlockImportError}
+};
 use crate::{
 	config::{Roles, BoxFinalityProofRequestBuilder},
-	message::{self, generic::FinalityProofRequest, BlockAttributes, BlockRequest, BlockResponse, FinalityProofResponse},
+	message::{
+		BlockAnnounce,
```

Comment:
better to wrap when the line limit exceeds.

Reply:
Why?

Reply:
consistency with the rest of substrate -- though this (afaik) is not strictly in our style guide. Usually what I do (and I think it is sort of the implicit styleguide):

```
use crate::{
     top_level_stuff_1, top_level_stuff_2, top_level_stuff_3, top_level_stuff_wrap_if_needed,
     inner_module_1::{stuff, bar, Bazinga},
     inner_module_2::{foo, kaboom, blah},
}
```

basically break once per sub-module but other than that only when needed.